scripts/find_cord19_length_outliers.py

```diff
@@ -0,0 +1,68 @@
+from pyserini.collection import pycollection
+import pandas as pd
+
+
+
+if __name__ == '__main__':
```

Comment:
don't hard code args, allow something like `-collection` and `-output` to specify input and output.

Reply:
done parser agrs